ui/package.json

```diff
@@ -37,7 +37,7 @@
     "@fortawesome/react-fontawesome": "^0.2.0",
     "@openapitools/openapi-generator-cli": "^2.7.0",
     "@terra-ui-packages/components": "0.0.4",
-    "@terra-ui-packages/core-utils": "0.0.5",
+    "@terra-ui-packages/core-utils": "0.0.6",
```

Comment:
No change in functionality, but adds some helpful type checks to prevent mistakes as [fixed here](https://github.com/all-of-us/workbench/pull/7940).